src/libs/freetype/lv_freetype_image.c

```diff
@@ -100,6 +100,11 @@ static const void * freetype_get_glyph_bitmap_cb(lv_font_glyph_dsc_t * g_dsc, lv
     };
 
     lv_cache_entry_t * entry = lv_cache_acquire_or_create(cache, &search_key, dsc);
+    if(entry == NULL) {
+        LV_LOG_ERROR("glyph bitmap lookup failed for glyph_index = 0x%" LV_PRIx32, (uint32_t)glyph_index);
```

Comment:
Let's use `%u` directly. Since FT_UInit is `unsigned int`.

Reply:
Don't worry, the `FT_Uint` is put into the `gid` from within the module. It put `FT_Uint` into the `gid` before, and then took it out again, so it's not a problem.